### PR TITLE
Explanation component test added

### LINKED ISSUE #211: specify annotation-explanation component
## Purpose

Show me an explanation text for what I am supposed to do during the annotation workflow of a specific category, so that I can successfully complete the activity.

## Use cases and input / output

- For the category I am currently annotating, display an explanation text to me in the user interface
  - input:
    - explanation text for the current category (plain string?)
  - output:
- I can click a UI element to collapse / hide the explanation so that I have more screen real estate
  - input and also output:
    - the open / closed state of the explanation component for the current category

## Internal state

- we could also have the open/closed state be stored in the component, but then it would reset every time we navigate away and back to the component.

## Ideas

- the open / closed state could also be something we store globally for all explanation components?
- the explanation component could have the ability to scroll if it exceeds a certain size
- we could also have this be a sidebar element in order to save more vertical screen real estate
- the side-bar could slide in from the side or even be a modal

### PR DESCRIPTION
**Pull request by surchs**
- added component test to illustrate prop behavior minor refactoring:
- use uuid package to ensure random ID instead of passing unique ID as prop
- minor refactor of variable names

closes #211

### CODE REVIEW

File: components/annot-explanation.vue [hunk 1]
```diff
@@ -47,11 +42,11 @@
 
             return {
 
-                // Text for UI elements
-                uiText: {
+                cardTitle: "Explanation",
```
Comment (jarmoza) [inline components/annot-explanation.vue:45]: This is cleaner, but inconsistent with how we've organized UI text in other components - the strings of which are currently contained in a `uiText` object.

Unless we are planning on moving these strings to the store, I suggest going back to `uiText.cardTitle`.

File: cypress/component/annotation-explanation.cy.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import annotExplanation from "~/components/annot-explanation";
+
+describe('explanation', () => {
```
Comment (jarmoza) [inline cypress/component/annotation-explanation.cy.js:3]: Just a small change to maintain consistent quotation characters throughout. We've gone with double quotation marks elsewhere.

**Review by jarmoza — CHANGES_REQUESTED** [review]
Just two small change requests. Otherwise, feel free to rebase afterwards!

File: components/annot-explanation.vue (hunk 1 above)
Comment (surchs) [reply components/annot-explanation.vue:45]: you're right. Makes sense to me

File: cypress/component/annotation-explanation.cy.js (hunk 2 above)
Comment (surchs) [reply cypress/component/annotation-explanation.cy.js:3]: Thanks, good catch. Not sure why this wasn't flagged by the precommit linter.

**Comment by surchs** [comment]
@jarmoza, I worked in both of your comments. If you're good, I'd merge this one with a squash commit because the style stuff doesn't need to be in the history.

**Review by jarmoza — APPROVED** [review]
Looks good, @surchs! Go for the squash merge.

**Pull request merged**